self-tests.yml

```diff
@@ -593,6 +602,11 @@ tasks:
     commands:
       - func: get-project
       - func: verify-agent-version-update
+  - name: verify-merge-function-update
```

Comment:
I think we should add `tags: ["linter"]` here and to `verify-agent-version-update`. Then at line 655 instead of adding those tasks individually to the build variant we can say `-name: .linter` and Evergreen will include any tasks that have the linter tag. We can also use this to update the PR definitions on the project page. Let me know if you have a question about this.

Reply:
done. 
 I wasn't going to update the project page until after it's merged so I might ask about it later!